modules/ppcp-settings/docs/merchant-post-onboarding.md

```diff
@@ -0,0 +1,414 @@
+- **Environment**: Production vs Sandbox determination
+- **Connection Status**: Boolean flag indicating active connection
+- **Account Type**: Business vs Personal seller classification
+
+### Settings Applied from Onboarding
+
+The onboarding preferences are applied as permanent settings:
+
+1. **Payment Method Enablement**: Based on `accept_card_payments` choice
+2. **Account Configuration**: Business vs personal account setup
+3. **Feature Activation**: Subscription support, vaulting, etc.
+4. **Gateway Registration**: WooCommerce payment gateway activation
+
+## Operational Differences: Connected vs Onboarding
+
+### During Onboarding
+- Limited to configuration flags based on country/plugin detection
+- No PayPal API access for merchant-specific data
+- Uses estimated capabilities for flow determination
+- Stores temporary preferences in `OnboardingProfile`
+
+### After Connection
+- Full PayPal API access with merchant authentication
+- Real-time capability detection from PayPal account
+- Access to merchant-specific features and limitations
+- Persistent settings in `GeneralSettings` and gateway configuration
+
+## API Integration Post-Connection
+
+### Merchant Information Retrieval
+
+Connected merchants can access PayPal's merchant APIs:
+
+1. **Account Information**: Real-time merchant details and capabilities
+2. **Transaction Processing**: Order creation, payment capture, refunds
+3. **Webhook Management**: Event notifications for payment updates
+4. **Feature Detection**: Dynamic capability queries
+
+### Settings Management
+
+The settings interface transitions from onboarding wizard to full configuration:
+
+- **Connection Tab**: Shows connected account details
+- **Payment Methods**: Enable/disable based on merchant capabilities
+- **Advanced Settings**: Country and account-specific options
+- **Troubleshooting**: Connection status and diagnostic tools
+
+### Post-Onboarding Capability Detection
+
+After successful merchant connection, the system can determine actual save payment capabilities through multiple approaches:
+
+#### Centralized Capability Service (Recommended)
+
+**File:** `modules/ppcp-settings/src/Service/MerchantCapabilities.php`
+
+The `MerchantCapabilities` service provides a unified, connection-aware approach to checking merchant capabilities:
+
+```php
+class MerchantCapabilities {
+    private ConnectionState $connection_state;
+    private DCCProductStatus $dcc_product_status;
+    private ReferenceTransactionStatus $reference_transaction_status;
+
+    public function can_save_paypal_methods(): bool {
+        if ($this->connection_state->is_connected()) {
+            return $this->reference_transaction_status->reference_transaction_enabled();
+        }
+        return false;
+    }
+
+    public function can_save_credit_cards(): bool {
+        if ($this->connection_state->is_connected()) {
+            return $this->dcc_product_status->is_active();
+        }
+        return false;
+    }
+}
+```
```

Comment:
Suggestion: Merge this class with the existing `MerchantDetails` class, which lives in the `ppcp-wc-gateway` module and already provides related capability checks